File: E-EstateV2 API/E-EstateV2 API/Models/LocalLabor.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace E_EstateV2_API.Models
{
    public class LocalLabor
    {
        [Key]
        public int Id { get; set; }
        public string monthYear { get; set; }
        public int totalWorker { get; set; }
        public string createdBy { get; set; }
        public DateTime createdDate { get; set; }
        public string updatedBy { get; set; }
        public DateTime updatedDate { get; set; }

        [ForeignKey("LaborTypeId")]
        public int laborTypeId { get; set; }
        public LaborType laborType { get; set; }

        [ForeignKey("EstateId")]
        public int estateId { get; set; }
        public Estate Estate { get; set; }
    }
}

File: E-EstateV2/src/app/_interface/localLabor.ts
import { LaborInformation } from "./laborInformation"

export interface LocalLabor {
    id: number
    monthYear: string | null
    totalWorker: number | null
    createdBy: string
    createdDate: Date
    updatedBy: string
    updatedDate: Date
    laborTypeId: number
    laborTypeName: string
    laborType: LaborInformation[]
    estateId: number
    totalLaborWorker: number
}